src/NUnitFramework/framework/Internal/Results/TestResult.cs

```diff
@@ -52,11 +54,23 @@ public abstract class TestResult : ITestResult
         /// </summary>
         internal const double MIN_DURATION = 0.000001d;
 
-//        static Logger log = InternalTrace.GetLogger("TestResult");
+        /// <summary>
+        /// Empty child queue
+        /// </summary>
+        protected static readonly ConcurrentQueue<ITestResult> EmptyChildQueue = new ConcurrentQueue<ITestResult> ();
+
+        //        static Logger log = InternalTrace.GetLogger("TestResult");
 
         private StringBuilder _output = new StringBuilder();
         private double _duration;
 
+        /// <summary>
+        /// Aggregate assertion count
+        /// </summary>
+        protected int InternalAssertCount;
```

Comment:
Sorry, I see the reason for ResultState below. I don't see the need for AssertCount. At a minimum, the backing field should be named correctly.